test/language/comments/hashbang-multi-line-comment.js

```diff
@@ -0,0 +1,15 @@
+#!/*
+these characters should not be considered within a comment
+*/
+/*---
+esid: pending
+description: >
+    Hashbang comments should not interpret multi-line comments.
+info: |
+    HashbangComment::
+      #! SingleLineCommentChars[opt]
+
+negative:
+  phase: parse
+  type: SyntaxError
+---*/
```

Comment:
lol, this test is so fun!

please add the `flags: [raw]` so test runners won't add harness files